pkg/services/authn/clients/ext_jwt.go

```diff
@@ -145,6 +144,8 @@ func (s *ExtendedJWT) authenticateAsUser(
 
 func (s *ExtendedJWT) authenticateAsService(claims *authlib.Claims[authlib.AccessTokenClaims]) (*authn.Identity, error) {
 	// Allow access tokens with that has a wildcard namespace or a namespace matching this instance.
+	// TODO: this is going to be a problem for service layer because it will need to send in *
+	// for the namespace when getting cluster scoped resources
 	if allowedNamespace := s.namespaceMapper(s.getDefaultOrgID()); !claims.Rest.NamespaceMatches(allowedNamespace) {
```

Comment:
Tricky bit. Need to differentiate between running modes and only apply this to embedded API Server / standalone Grafana. Is there any easy differentiator that I am missing?

Reply:
Not a concern at the moment. The auth code written by the auth team okay's '*' for use in ST Grafana. If that is a decision that impacts us badly, we can visit this again.